proceeding/text/data.tex

```diff
@@ -22,7 +22,7 @@ \section{Data}
 
   \item SNRcat
 
-  \item GeTeV Catalogue
+  \item gamma-cat open TeV catalog (replacing GeTeV Catalogue)
```

Comment:
Remove `(replacing GeTeV Catalogue)`.
We'll just do it, it was never in a released version of the website, no need to mention it.